docs/_guide/local-docs.md

```diff
@@ -0,0 +1,48 @@
+There are two parts to the fuels-ts repository documentation
+
+## Part One: Typedoc
+[typedoc](https://typedoc.org/) does the work of gathering all types, functions, classes, etc. from all of our source code. It will read through the `packages` folder and collect all that information and convert it into a markdown file, modifying the contents of [`docs/packages`](https://github.com/FuelLabs/fuels-ts/tree/master/docs/packages).

```

Comment:
```suggestion
[typedoc](https://typedoc.org/) gathers all types, functions, classes, etc., from our source code. It will read through the `packages` folder, collect all that information, and convert it into a markdown file, modifying the contents of [`docs/packages`](https://github.com/FuelLabs/fuels-ts/tree/master/docs/packages).

```

Reply:
Sure, disregard the other one; it was mostly punctuation stuff. We can do that later 😄